chef_master/source/config_rb_client.rst

```diff
@@ -52,7 +52,10 @@ This configuration file has the following settings:
 
 ``checksum_path``
    The location in which checksum files are stored. These are used to validate individual cookbook files, such as recipes. The checksum itself is stored in the Chef server database and is then compared to a file in the checksum path that has a filename identical to the checksum.
-
+   
+``chef_guid``
+   The node uuid used by automate. Setting this allows the node uuid to be specified, and can be carried across instances of a node. 
```

Comment:
The node uuid sent to Chef Automate as specified by `RFC077 <https://github.com/chef/chef-rfc/blob/master/rfc077-mode-agnostic-data-collection.md>`.